arangod/Cluster/ClusterFeature.cpp

```diff
@@ -497,6 +497,9 @@ void ClusterFeature::unprepare() {
   AgencyCommManager::MANAGER->stop();
 
   ClusterInfo::cleanup();
+
+  // no more tcp requests from now on.
+  ApplicationServer::server->beginUnprepare();
```

Comment:
I think we shouldn't do it like this.
Why is a single feature responsible for unpreparing the entire ApplicationServer?
What if the ClusterFeature is disabled?